packages/react-components/react-drawer/stories/src/Drawer/DrawerPosition.stories.tsx

```diff
@@ -20,34 +20,20 @@ const useStyles = makeStyles({
   },
 });
 
-const setTitle = (position: DrawerProps['position']) => {
-  switch (position) {
-    case 'start':
-      return 'Left';
-
-    case 'end':
-      return 'Right';
-
-    case 'bottom':
-      return 'Bottom';
-
-    default:
-      return undefined;
-  }
-};
+const capitalize = (str?: string) => str && str.charAt(0).toUpperCase() + str.slice(1);
```

Comment:
this api seems flawed, why would one allow to call capitalize on undefined value ?